src/main/java/com/cms/claimmanagement/insuranceCompany/surveyor/surveyReport/controller/ReportController.java

```diff
@@ -0,0 +1,31 @@
+package com.cms.claimmanagement.insuranceCompany.surveyor.surveyReport.controller;
+
+import com.cms.claimmanagement.insuranceCompany.surveyor.surveyReport.SurveyReportDTO;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/surveyReports")
+public class ReportController {
+
+    private final SurveyReportService surveyReportService;
+    @Autowired
+    public ReportController(SurveyReportService surveyReportService) {
+        this.surveyReportService = surveyReportService;
+    }
+    @PostMapping( consumes = {"application/json"})
+    public String saveReport(final @RequestBody SurveyReportDTO surveyReportDTO){
+        return surveyReportService.saveReport(surveyReportDTO);
+    }
+    @GetMapping(path = "/{claimId}")
```

Comment:
same as before, the path param is used with the id of the resource. If you are filtering by claimId, which is the id of other resource, maybe you could use a queryParam as a filter. 

/v1/surveyReports/1 - returns the surveyReport with id equals 1 
/v1/surveyReports?claimId=1 - returns surveyReport for the claim with id equals 1.